### PR TITLE
React to HttpAbstractions change: No features in `.Internal` namespace

### PR DESCRIPTION
**Pull request by dougbu**
- see issue aspnet/HttpAbstractions#561 and pull aspnet/HttpAbstractions#589

### CODE REVIEW

File: src/Microsoft.AspNetCore.CookiePolicy/CookiePolicyMiddleware.cs [hunk 1]
```diff
@@ -5,6 +5,7 @@
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http.Features;
 using Microsoft.AspNetCore.Http.Features.Internal;
```
Comment (Tratcher) [inline src/Microsoft.AspNetCore.CookiePolicy/CookiePolicyMiddleware.cs]: What is this one for?

**Comment by Tratcher** [comment]
:shipit:

File: src/Microsoft.AspNetCore.CookiePolicy/CookiePolicyMiddleware.cs (hunk 1 above)
Comment (dougbu) [inline src/Microsoft.AspNetCore.CookiePolicy/CookiePolicyMiddleware.cs]: See [line 30](https://github.com/aspnet/Security/blob/dougbu/feature.interfaces.561/src/Microsoft.AspNetCore.CookiePolicy/CookiePolicyMiddleware.cs#L30) in this file. The `class` references both `IResponseCookiesFeature` and the concrete `ResponseCookiesFeature`. (As I [mentioned](https://github.com/aspnet/HttpAbstractions/issues/549#issuecomment-198424718) in aspnet/HttpAbstractions#549, many of our repos use supposedly-internal classes and interfaces from HttpAbstractions.)

**Comment by dougbu** [comment]
9361960

**Pull request merged**